kyuubi-common/src/main/scala/org/apache/kyuubi/engine/EngineType.scala

```diff
@@ -23,5 +23,5 @@ package org.apache.kyuubi.engine
 object EngineType extends Enumeration {
   type EngineType = Value
 
-  val SPARK_SQL, FLINK_SQL, TRINO = Value
+  val SPARK_SQL, FLINK_SQL, TRINO, HIVE_SQL = Value
```

Comment:
I think we should use `HIVE` here, for Flink and Spark, they provide user interfaces not only SQL, for Trino and Hive, only SQL, `HIVE_SQL` is quite redundant.

Reply:
Oh, I see we already named the module `kyuubi-hive-sql-engine`, `kyuubi-trino-sql-engine` looks abnormal then...

Reply:
So, whether to rename them anywhere or keep them.

Reply:
cc @yaooqinn